android/java/org/chromium/chrome/browser/crypto_wallet/fragments/UnlockWalletFragment.java

```diff
@@ -201,8 +204,7 @@ private void showPasswordRelatedControls() {
         mUnlockButton.setVisibility(View.VISIBLE);
         mUnlockWalletRestoreButton.setVisibility(View.VISIBLE);
         mUnlockWalletTitle.setVisibility(View.VISIBLE);
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P
```

Comment:
the same about biometric and API <=28.

Reply:
https://github.com/brave/brave-core/pull/23127/files#r1574378595